useScroll/ScrollButtons.js

```diff
@@ -34,212 +32,81 @@ const
 /**
  * An Agate-styled scroll buttons. It is used in [Scrollbar]{@link agate/useScroll.Scrollbar}.
```

Comment:
Would you please update JS doc here? like `A custom hook that returns Agate-themed scroll buttons behavior`?